src/caja-desktop-window.c

```diff
@@ -174,10 +191,27 @@ caja_desktop_window_new (CajaApplication *application,
     int width_request, height_request;
     int scale;
 
-    scale = gdk_window_get_scale_factor (gdk_screen_get_root_window (screen));
-
-    width_request = WidthOfScreen (gdk_x11_screen_get_xscreen (screen)) / scale;
-    height_request = HeightOfScreen (gdk_x11_screen_get_xscreen (screen)) / scale;
+    GdkDisplay *display = gdk_screen_get_display (screen);
+    if  (GDK_IS_X11_DISPLAY (display))
```

Comment:
Too much spaces after the if.

Reply:
There and multiple other places in same file, all found and fixed